lambdaclass/mirra_backend
# Fix default bounty crashing game

> [!warning]
> Should be merged alongside [This client branch](https://github.com/lambdaclass/champions_of_mirra/pull/2008)

## Motivation

If you don't pick any bounty and complete it in the game the client crashes, we need to send the selected bounty to the client

## Summary of changes

- Added a new selected_bounty message when picking a default bounty or when the user picks one

## How to test it?

Start a match in [This client branch](https://github.com/lambdaclass/champions_of_mirra/pull/2008), if you don't pick any bounty and complete it the game should keep running

## Checklist
